pkg/remote/message.go

```diff
@@ -185,7 +215,7 @@ func (m *message) NewData(method string) (ok bool) {
 	if m.data != nil {
 		return false
 	}
-	if mt := m.svcInfo.MethodInfo(method); mt != nil {
+	if mt := m.methodSvcMap[method].MethodInfo(method); mt != nil {
```

Comment:
TODO: need to fix. when method names are conflicting, non-fallback service cannot be accessed.